google/cloud/aiplatform/training_jobs.py

```diff
@@ -2673,6 +2673,459 @@ def _model_upload_fail_string(self) -> str:
+                hours i.e. 1,000 value in this field means 1 node hour.
+                The training cost of the model will not exceed this budget. The final
+                cost will be attempted to be close to the budget, though may end up
+                being (even) noticeably smaller - at the backend's discretion. This
+                especially may happen when further model training ceases to provide
+                any improvements.
+                If the budget is set to a value known to be insufficient to train a
+                Model for the given training set, the training won't be attempted and
+                will error.
+                The minimum value is 1000 and the maximum is 72000.
+            model_display_name (str):
+                Optional. If the script produces a managed AI Platform Model. The display name of
+                the Model. The name can be up to 128 characters long and can be consist
+                of any UTF-8 characters.
+
+                If not provided upon creation, the job's display_name is used.
+            sync (bool):
+                Whether to execute this method synchronously. If False, this method
+                will be executed in concurrent Future and any downstream object will
+                be immediately returned and synced when the Future has completed.
+        Returns:
+            model: The trained AI Platform Model resource or None if training did not
+                produce an AI Platform Model.
+        """
+
+        training_task_definition = schema.training_job.definition.automl_forecasting
+
+        training_task_inputs_dict = {
+            # required inputs
+            "targetColumn": target_column,
+            "timeColumn": time_column,
+            "timeSeriesIdentifierColumn": time_series_identifier_column,
+            "staticColumns": static_columns,
+            "timeVariantPastOnlyColumns": time_variant_past_only_columns,
+            "timeVariantPastAndFutureColumns": time_variant_past_and_future_columns,
+            "forecastWindowEnd": forecast_window_end,
+            "period": {"unit": period_unit, "quantity": period_count},
+            "transformations": self._column_transformations,
+            "trainBudgetMilliNodeHours": budget_milli_node_hours,
+            # optional inputs
+            "weightColumn": weight_column,
+            "forecastWindowStart": forecast_window_start,
+            "pastHorizon": past_horizon,
+            "quantiles": quantiles,
+            "validationOptions": validation_options,
+            "optimizationObjective": self._optimization_objective,
+        }
+
+        final_export_eval_bq_uri = export_evaluated_data_items_bigquery_destination_uri
+        if final_export_eval_bq_uri and not final_export_eval_bq_uri.startswith(
+            "bq://"
+        ):
+            final_export_eval_bq_uri = f"bq://{final_export_eval_bq_uri}"
+
+        if export_evaluated_data_items:
```

Comment:
@sasha-gitg Seems like `definition/automl_tabular_1.0.0.yaml` also defines `exportEvaluatedDataItemsConfig`. This is unimplemented for `AutoMLTabularTrainingJob`.

I'll add a bug to track this.